ui/app/src/views/MigrateView.tsx

```diff
@@ -148,18 +152,27 @@ function MigrateView() {
             {migrateMutation.error.message}
           </Alert>
         )}
-        {migrateMutation.isSuccess && (
-          <Stack direction={'row'} spacing={1}>
+        {!isLoading && data !== undefined && data !== null && migrateMutation.isSuccess && (
+          <Stack direction={'row'}>
```

Comment:
The best practice is to remove curly brackets when it's only containing a hardocded string 😄 Easier to read too